main/store/migrations/index.js

```diff
@@ -407,7 +407,66 @@ const migrations = {
     })
 
     return initial
-  }
+  },
+  21: initial => {
+    // add sepolia network information
+    if (!initial.main.networkPresets.ethereum[11155111]) {
```

Comment:
don't think we need this

Reply:
I think I tried it without and it didn't work.  Will try again

Reply:
the networks presets aren't even loaded from disk so the migration wouldnt be doing anything

Reply:
should we just delete them from the store altogether then?